sunpy/physics/solar_rotation.py

```diff
@@ -144,3 +144,70 @@ def mapcube_solar_derotate(mc, layer_index=0, clip=True, shift=None, **kwargs):
 
     # Apply the pixel shifts and return the mapcube
     return apply_shifts(mc, yshift_keep, xshift_keep, clip=clip)
+
+
+def mapsequence_solar_derotate(mc, layer_index=0, clip=True, shift=None, **kwargs):
```

Comment:
the mapcube version of this function should call this one to prevent the redefinition of the whole function.